### ISSUE TITLE
Permettre une recherche à facettes

### ISSUE THREAD

**Issue by agarrone** [datagouv/data.gouv.fr#1432]
Faire évoluer la recherche sur data.gouv.fr au profit d'une recherche à facettes notamment pour permettre aux besoins suivants : 
- Selection de plusieurs filtres en même temps 
- Affichage du nombre de résultats par facettes

**Comment by maudetes** [comment]
Est-ce qu'on veut clore ce ticket @ThibaudDauce ? Ou on veut rajouter le support des facettes pour les filtres avancés (et en particulier les organisations) ?

**Comment by agarrone** [comment]
Yes je pense qu'il nous faut le support des facettes pour les filtres avancés aussi.

On peut peut être rename ?

**Comment by ThibaudDauce** [comment]
Oui on veut ajouter ça, maintenant qu'on a intégré udata-search-service ça va être rapide à faire, je regarde lundi :-)

**Comment by ThibaudDauce** [comment]
@maudetes Qu'est-ce que tu imaginais pour la recherche des orgas. Est-ce qu'on continue de se baser sur les les résultats par défaut du suggest (et dans ce cas toutes les orgas n'ont pas de facets car ES nous retourne les 50 orgas avec le plus de documents pour la recherche en cours), ou alors on switch sur les 50 orgas justement par défaut (tant qu'on a pas de recherche d'orga) mais dans ce cas on se retrouve avec les Hauts de France en premier car ils ont 21000 datasets…

**Comment by streino** [comment]
Facettes sur orga ça nous intéresse aussi ! cc @abulte 

Tu peux définir le critère de tri si count n'est pas adapté : https://www.elastic.co/docs/reference/aggregations/search-aggregations-bucket-terms-aggregation#search-aggregations-bucket-terms-aggregation-order

**Comment by maudetes** [comment]
Ah je retrouve le ticket en question. Je remets ma proposition pour data.gouv.fr :
- tant qu'il n'y a aucun filtre, on utilise la liste issue du suggest
- dès qu'on a un filtre/une query, on passe sur les facettes
